docs/specification.md

```diff
@@ -78,11 +78,11 @@ Additionally, each minter is associated with its current minting allowance.
 Before describing the token contract's entrypoints we describe the `Safelist` interface.
 The syntax is: `entrypointName(argumentType)`.
 Required `Safelist` entrypoints:
-* `assertTransfers(list(pair (address :from) (address :to)))`
+* `assertTransfers(list(pair (address :from) (list(address) :tos)))`
```

Comment:
`from` and `tos` should be field annotations, not type annotations, according to https://gitlab.com/tzip/tzip/-/merge_requests/61/diffs.

Reply:
So that is why they included it, even with Ligo's issue with "from" keyword. It does not apply to field annotations.

I have made the change and also updated the spec. I have removed the TODO and NOTE completely as the interface matches with FA2 now.